pkg/dynamicinstrumentation/ditypes/arch_arm64.go

```diff
@@ -0,0 +1,10 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package ditypes
+
+const StackRegister = 29
```

Comment:
add a comment to explain this const, btw won't you have a name conflict with the same const in the amd64 file? (the build tags for those files are not mutually exclusive iiuc)

Reply:
The filename ending in arm64 vs amd64 function as build tags

Reply:
any specific reason to use a different convention than the build tags?

Reply:
We have to put them in separate files anyway which have to be named differently. This convention is used in a lot of places in the repo ([example](https://github.com/DataDog/datadog-agent/blob/grantseltzer/DEBUG-2982-multiple-dereference-support-2/pkg/security/secl/model/syscalls_linux_amd64.go))

Reply:
I think the standard way in system-probe is via build tags (there are indeed a few example in security pkg that don't follow that)

I think [this](https://github.com/DataDog/datadog-agent/blob/7fdeef9e7aa540542d504bbc07185459eb3ea83c/cmd/system-probe/modules/all_linux.go) is the standard for system-probe

Reply:
Not a problem, i've added build tags in the files as well.